ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -801,7 +804,7 @@ dial_liveness_readiness:
     periodSeconds: 15
     timeoutSeconds: 5
     failureThreshold: 2
-
+    
```

Comment:
I dont know if this is a tab or white spaces. Either way, please remove the extra spaces / tab.

Reply:
Removed Tab space.